lib/node_modules/@stdlib/math/base/special/cflipsign/benchmark/c/benchmark.c

```diff
@@ -105,7 +105,7 @@ double benchmark() {
 	for ( i = 0; i < ITERATIONS; i++ ) {
 		v = ( 1000.0*rand_double() ) - 500.0;
 		x = v + v*I;
-		y = stdlib_base_cflipsign( x, -v );
+		y = stdlib_base_flipsign( creal( x ), -v ) + stdlib_base_flipsign( cimag( x ), -v )*I;
```

Comment:
Instead of using `base_flipsign` here, I'd say we can just remove this benchmark file and its associated Makefile.